.github/workflows/release.yaml

```diff
@@ -14,7 +14,7 @@ jobs:
       - name: Set up Python
         uses: actions/setup-python@v4
         with:
-          python-version: "3.7"
```

Comment:
will we not need python 3.7 for mwaa?

Reply:
We are not using the lib on mwaa, only on the lambda now.